requirements.txt

```diff
@@ -22,7 +22,7 @@ extractcode==31.0.0
 extractcode-7z==16.5.210531
 extractcode-libarchive==3.5.1.210531
 fasteners==0.19
-fingerprints==1.2.3
+fingerprints==1.3.0
```

Comment:
We cannot upgrade `fingerprints` just yet, because they have removed support for python3.9, see https://github.com/opensanctions/fingerprints/blob/8eea92ff357e080c6c8bb0807352e7b5f83d9c14/pyproject.toml#L19C1-L19C28
This is not EOL yet: https://endoflife.date/python

Also why the tests are failing because we test and support python3.9.

Note also as mentioned in https://github.com/aboutcode-org/scancode-toolkit/issues/4493#issue-3272342439, fingerprints is now unmaintained.
So we would want to consider switching to rigour in the future.